docs/spec/src/integration/spec/5-lifecycle-phases.md

```diff
@@ -0,0 +1,169 @@
+    
+
+> Note that the length here is the length of the blob (power of 2), which is different from the payload_length encoded as part of the `PayloadHeader` in the `blob` itself (see the [encoding section](#encoding)).
+> 
+
+**PaymentHeader**
+
+The paymentHeader specifies how the blob dispersal to the network will be paid for. There are 2 main modes of payment, the permissionless pay-per-blob model and the permissioned reserved-bandwidth approach. See the [Payments](https://docs.eigenda.xyz/releases/payments) release doc for full details; we will only describe how to set these 4 fields here.
+
+```protobuf
+message PaymentHeader {
+  // The account ID of the disperser client. This should be a hex-encoded string of the ECSDA public key
+  // corresponding to the key used by the client to sign the BlobHeader.
+  string account_id = 1;
+  uint32 reservation_period = 2;
+  bytes cumulative_payment = 3;
+    // The salt is used to ensure that the payment header is intentionally unique.
+  uint32 salt = 4;
+}
+```
+
+Users who want to pay-per-blob need to set the cumulative_payment. Users who have already paid for reserved-bandwidth can set the reservation_period.
+
+An rpc call to the Disperser’s `GetPaymentState` method can be made to query the current state of an `account_id`. A client can query for this information on startup, cache it and then update it manually when making pay-per-blob payments, by keeping track of its current `cumulative_payment` to set it correctly for subsequent dispersals.
+
+The `salt` is needed for reserved bandwidth users: given that the `blob_header_hash` uniquely identifies a dispersal, if a dispersal ever fails (see next section), then to redisperse the same blob, a new unique `blob_header` must be created. If the `reservation_period` is still the same (currently set to 300 second intervals), then the salt must be increased (or randomly changed) to allow resubmitting the same `blob`.
+
+## Blob Dispersal
+
+The `DisperseBlob` method takes a `blob` and `blob_header` as input. Dispersal entails taking a blob, reed-solomon encoding it into chunks, dispersing those to the EigenDA nodes, retrieving their signatures, creating a `cert` from them, and returning that cert to the client. The disperser batches blob for a few seconds before dispersing them to nodes, so an entire dispersal process can take north of 10 seconds. For this reason, the API has been designed asynchronously with 2 relevant methods:
```

Comment:
The disperser batches *blobs*

Reply:
[176dd65](https://github.com/Layr-Labs/eigenda/pull/1439/commits/176dd652e57e14075b2e32d93af34488a4fff2d9)